admin/edit-user.html

```diff
@@ -3,10 +3,112 @@
 title: Add user, edit user, your account
 ---
 
-<div class="mw6 center mv6">
-  <h1>This will show states for:</h1>
-  <ul>
-    <li>Add user (only available to Admin user)</li>
-    <li>Edit user (only available to Admin user)</li>
-  </ul>
-</div>
+<main class="mw7 center ph2">
+  <section class="cf mt4 mb3 pa3 pa4-l bg-white shadow br3">
+    <h1 class="mb4 f4 fw6">Add User</h1>
+    <form>
+      <div class="flex flex-wrap items-end">
+        <div class="w-100 w-50-ns mb4 pr1-ns">
+          <label for="name" class="db mb2 fw6">
+            Name *
+          </label>
+          <input id="name" type="text" class="w-100 pa3 br2 b--black-10" />
+        </div>
+      </div>
+      <div class="flex flex-wrap items-end">
+        <div class="w-100 w-50-ns mb4 pr1-ns">
+          <label for="email" class="db mb2 fw6">
+            Email *
+          </label>
+          <input id="email" type="text" class="w-100 pa3 br2 b--black-10" />
+        </div>
+      </div>
+      <div class="flex flex-wrap items-end">
+        <div class="w-100 w-50-ns mb4 pr1-ns">
+          <label for="role" class="db mb2 fw6">
+            Role
+          </label>
+          <ul class="list pl0 ml0 center mw6 ba bw1 b--black-10 br2">
+            <li class="ph3 pv3 bb bw1 b--black-10">
+              <div class="pb2">
+                <input
+                  type="radio"
+                  id="search"
+                  name="role"
+                  value="search"
+                  class="v-top"
+                  checked
+                />
+                <label for="search" class="pl1 fw6">Search</label>
+              </div>
+              <div>
+                <ul class="pl3">
+                  <li class="mb1">&bull;&nbsp;Can search records</li>
+                </ul>
+              </div>
+            </li>
+            <li class="ph3 pv3">
+              <div class="pb2">
+                <input
+                  type="radio"
+                  id="admin"
+                  name="role"
+                  value="admin"
+                  class="v-top"
+                />
+                <label for="admin" class="pl1 fw6">Admin</label>
+              </div>
+              <div>
+                <ul class="pl3">
+                  <li class="mb1">&bull;&nbsp;Can search records</li>
+                  <li class="mb1">&bull;&nbsp;Can manage users and groups</li>
+                </ul>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="flex flex-wrap items-end">
+        <div class="w-100 w-50-ns mb4 pr1-ns">
+          <label for="group" class="db mb2 fw6">
+            Group
+          </label>
+          <select
+            id="group"
+            class="w-100 pa3 br2 bw1 b--black-10 input-reset bg-white black-60 indent"
```

Comment:
Why did you use an indent here?

Reply:
.indent class was stray, have now removed